Task_TheFlexGame/index.css

```diff
@@ -1 +1,60 @@
-/* Start coding below for FLEX GAME TASK*/
+/* Start coding below for FLEX GAME TASK*/
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.container{
+    display: flex;
+    justify-content: center;
+    gap: 1rem;
+    height: 100vh;
+    width: 100vw;
+    font-size: x-large;
+    font-weight: bold;
+}
+
+.menu{
+    display: flex;
+    background-color: #6249C3;
+    color: aliceblue;
+    flex-basis: 20%;
+    gap: 1rem;
+    padding: 25px;
+    flex-direction: column;

```

Comment:
Consider adding a bit more spacing between the properties in the .menu class for better readability.